### DISCUSSION TITLE
GUILD_ID is not defined

### DISCUSSION THREAD

**Discussion by isis1234** [discordjs/discord.js#8448] [Q&A] [answered]
As a topic question, I following [the introduction](https://github.com/discordjs/discord.js) about `Register a slash command against the Discord API`, I can found `CLIENT_ID `and `token`, but where is the `GUILD_ID `and how can I found it?

**Comment by Jackietkfrost** [comment, marked as answer]
Hey, to get a guild ID, you need to set your discord into developer mode, and then right click the server icon, and click copy ID. Hope that helps.
